WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -70,7 +70,7 @@ private extension CartView {
 
 #if DEBUG
 #Preview {
-    CartView(viewModel: PointOfSaleDashboardViewModel(products: POSProductFactory.makeFakeProducts(),
+    CartView(viewModel: PointOfSaleDashboardViewModel(products: POSProductFactory.makeFakeProducts(currencySettings: ServiceLocator.currencySettings),
```

Comment:
> nit: I'd suggest passing a placeholder value like `.init()` so that this POS file isn't dependent on `ServiceLocator` (even though it's in preview it would make any potential move a separate place easier in the future)

How exactly do you mean?
To add a default parameter in `makeFakeProducts` or?

Reply:
Sorry I wasn't very clear about what parameter to pass a placeholder, I meant `currencySettings: ServiceLocator.currencySettings` --> `currencySettings: .init()`.

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/12852